Fluid/Ast/IncludeStatement.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Microsoft.Extensions.FileProviders;
+
+namespace Fluid.Ast
+{
+    public class IncludeStatement : Statement
+    {
+        private static readonly IFileProvider _blankFileProvider = new NullFileProvider();
+
+        public IncludeStatement(Expression path)
+        {
+            Path = path;
+        }
+
+        public Expression Path { get; }
+
+        public override async Task<Completion> WriteToAsync(TextWriter writer, TextEncoder encoder, TemplateContext context)
+        {
+            var relativeLayoutPath = (await Path.EvaluateAsync(context)).ToStringValue();
+            var currentViewPath = context.AmbientValues["ViewPath"] as string;
```

Comment:
I think it's not necessary in this case, and actually wrong. Because for an include the file will be specified from the root of the file provider. It is different than in the view engine where we have more conventions, like requiring to have the views provided by the LocationExpander.

For instance, if the dev wants the partials in a `/partials` folder then he will provide a file provider with a root there.